test/unit/services/instanceAdapter_spec.js

```diff
@@ -49,33 +67,181 @@ describe("The InstanceAdapter class ", function () {
 
 	});
 
-	describe("when the mapper fails to create a new model", function () {
-		var result;
-		var mapperStub;
+	describe("Getting an instance", function () {
+		describe("with a valid instanceId", function () {
+			var VALID_INSTANCE_ID;
+			var DEFAULT_TYPE = "t2.micro";
+			var DEFAULT_AMI  = "ami-d05e75b8";
+			var ID_REGEX     = /[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/;
+			var instances    = new InstanceAdapter();
+
+			before(function () {
+				instances.createInstance()
+				.then(function (response) {
+					expect(response).to.be.an.instanceOf(Instance);
+					expect(response.id).to.match(ID_REGEX);
+					expect(response.type).to.equal(DEFAULT_TYPE);
+					expect(response.ami).to.equal(DEFAULT_AMI);
```

Comment:
What is the thought behind having these assertions here? If the `createInstance()` function works correctly, can't we just assume that this information is correct?